linkerd/app/core/src/metrics.rs

```diff
@@ -9,14 +9,15 @@
 pub use crate::transport::labels::{TargetAddr, TlsAccept};
 use crate::{
     classify::Class,
-    control, http_metrics, opencensus, opentelemetry, profiles, stack_metrics,
-    svc::Param,
-    tls,
+    control, http_metrics, opencensus, opentelemetry, profiles, stack_metrics, tls,
     transport::{self, labels::TlsConnect},
 };
 use linkerd_addr::Addr;
 pub use linkerd_metrics::*;
+use linkerd_proxy_api_resolve::Metadata;
 use linkerd_proxy_server_policy as policy;
+use linkerd_stack::Param;
```

Comment:
nit: We tend to use the local reexports like svc::Param (as before) and proxy::api_resolve::Metadata